pkgs/tools/misc/handlr-regex/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub, shared-mime-info, libiconv, installShellFiles }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "handlr-regex";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "Anomalocaridid";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-X0j62Ntu/ouBVm09iKxU3pps8mbL5V5gA65Maa4b0AY=";
+  };
+
+  cargoSha256 = "sha256-byR7CM876z5tAXmbcUfI0CnJrc/D6CxfjBJhuJMSFmg=";
+
+  nativeBuildInputs = [ installShellFiles shared-mime-info ];
+  buildInputs = lib.optionals stdenv.isDarwin [ libiconv ];
```

```suggestion
  buildInputs = [ libiconv ];
```
would be the same since libiconv is null on linux

---

Just in case it's ever relevant, that part was copied verbatim from the module for the original handlr. So the same redundant check is still present there.